### DISCUSSION TITLE
Adding a link to this Toolkit in Canada.ca's design system?

### DISCUSSION THREAD

**Discussion by quidampepin** [gc-da11yn/gc-da11yn.github.io#34] [General (Open Discussions)]
Hey - Awesome work! 

Would there be a problem with us (DTO) linking to that toolkit on https://design.canada.ca/designing-content.html#accessibility? 

Merci!

**Comment by shawnthompson** [comment]
Not at all. Link away. We would like to keep you folks in discussions on design decisions we make.

**Reply by quidampepin** [reply]
I added the link. Don't hesitate to tag me in a discussion about design decisions. :-)
